lib/connection.h

```diff
@@ -753,5 +737,11 @@ namespace QMatrixClient
             static room_factory_t _roomFactory;
             static user_factory_t _userFactory;
     };
+
+    inline QDebug operator<<(QDebug dbg, const Connection::SupportedRoomVersion& v)
+    {
+        return dbg.nospace() << v.id << '/' << v.status;
+    }
```

Comment:
Please get it back as a friend function inside `SupportedRoomVersion`, for the sake of code locality. Using `friend`  is a way to define inside classes logically-close functions that are not class/structure members (not only to allow them access to non-public members).

Reply:
Ok